src/java.base/share/classes/java/util/HashMap.java

```diff
@@ -2545,4 +2545,34 @@ else if ((xpl = xp.left) == x) {
         }
     }
 
+    /**
+     * Calculate initial capacity for HashMap based classes, from expected size.
```

Comment:
This method should say it is using a default load factor  (0.75).
I'm not sure how often anyone creates a map with a non-default load factor
but that might suggest a more general API pattern either in the supporting calculation
or the public factories.

Reply:
> This method should say it is using a default load factor (0.75).

@RogerRiggs done.